src/vs/workbench/browser/parts/compositePart.ts

```diff
@@ -426,6 +428,8 @@ export abstract class CompositePart<T extends Composite> extends Part {
 			'class': ['composite', 'title']
 		});
 
+		this._register(DOM.addDisposableListener(titleArea.getHTMLElement(), DOM.EventType.CONTEXT_MENU, e => this.onContextMenu(new StandardMouseEvent(e))));
```

Comment:
I am not a user of Builders, instead DOM apis. Hence did not know above pattern. Looks simple and short to use with builder. Thanks.